qa/workunits/rbd/rbd_mirror_ha.sh

```diff
@@ -0,0 +1,207 @@
+#!/bin/sh
```

Comment:
@trociny Hmm -- perhaps what we really just need (long term) is a "thrasher" task like the ones for OSDs/MDSs. I'll create a separate tracker ticket for that.